runbooks/workflow/payments/add_agency_dashboard_data_source.md

```diff
@@ -0,0 +1,190 @@
+
+1. To begin, create a new service account for the agency within the Google Cloud Platform console, which will be used to allow Metabase read-access to the agency's payments data.
+
+- Navigate to: <https://console.cloud.google.com/iam-admin/serviceaccounts> in the `cal-itp-data-infra` project
+- Select `+ Create Service Account` in the top-center of the page
+- Populate the `Service account ID` field using the convention: `[agency-name]-payments-user`, then select `Create and Continue`
+- Within the `Grant this service account access to the project` section, assign the role `Agency Payments Service Reader`
+- Select `Done`
+
+2. Download the service account key for the service account you've just created
+
+- After selecting `Done` in the previous section you'll be returned to the list of existing service accounts. Click into the service account that you just created.
+- Select `Keys` from the top-center of the page and then select the `Add Key` dropdown. Choose the `Create new key` selection within that.
+- Keep the default key type `JSON` and select `Create`
+- This will download a JSON copy of the service accout key to your local environment, which will be used in later steps within Metabase
+
+3. Open a new branch in the [cal-itp/data-infra Github repository](https://github.com/cal-itp/data-infra) and edit the [create_row_access_policy macro](https://github.com/cal-itp/data-infra/blob/main/warehouse/macros/create_row_access_policy.sql)
+   **Permissions needed**: Member of the [cal-itp Github organization](https://github.com/cal-itp)
+
+Duplicate an existing row access policy within the file and append to the bottom, before the `{% endmacro %}` text.
+
+The contents of the policy you're duplicating should look like this:
+
+```
+{{ create_row_access_policy(
+    filter_column = 'participant_id',
+    filter_value = '[agency-name]',
+    principals = ['serviceAccount:`agency-name`-payments-user@cal-itp-data-infra.iam.gserviceaccount.com']
+) }};
+```
+
+Substitute the following fields with the appropriate information for the agency that you are adding:
+
+- `filter_value` which is the Littlepay `participant_id` for the agency
+- `principals` which is the email address for the service account that was created in step #1. You can simply subsitute the agency name as used in that step as opposed to updating the whole string.
+
+Open a PR in Github and merge these changes. If you'd like access to the results of this policy before the next time the `transform_warehouse` DAG is run, you will need to run it manually. To do this, you should trigger the DAG with a selector [as described in the README for the DAG task](https://github.com/cal-itp/data-infra/tree/main/airflow/dags/transform_warehouse). Use selector: `{"dbt_select": "models/mart/payments"}`.
```

Comment:
Suggest "to" rather than "and" in the first sentence in order to implicitly clarify that people generally shouldn't fully control the PR process without input/review from others.